src/Microsoft.DotNet.Wpf/src/PresentationCore/System/Windows/Automation/Peers/AutomationPeer.cs

```diff
@@ -652,6 +652,9 @@ virtual protected string GetLocalizedControlTypeCore()
         ///
         abstract protected bool IsControlElementCore();
 
+        ///
+        abstract protected bool IsDialogCore();
```

Comment:
We can't add abstract members to a class that has already shipped - it breaks subclasses.  Instead make this virtual, and return false here.

Reply:
Aha!  That's what the first baseline error is trying to tell you.   Sorry I didn't make that connection earlier.

Reply:
Yes, that solved the problem. I tried to follow the pattern of all the boolean properties being abstract.